.gitignore

```diff
@@ -68,3 +68,4 @@ generated_docs/
 TODO.md
 
 packages/website/public/bundle*
+transpiled/
```

Comment:
Can we compile to the `lib` folder as we do in all other packages?

Reply:
Yeah, I'd prefer that. Also, let's make sure it's the `lib` folder nested under `packages/contracts`.